src/main/java/com/google/sps/data/PlaceReviews.java

```diff
@@ -16,11 +16,9 @@
  */
 public class PlaceReviews {
 
-  public final String place_id;
-  public List<Comment> reviews;
-  private List<String> reviewers;
+  private final String place_id;
+  private ArrayList<Comment> reviews;
```

Comment:
This could instead be
`List<String> comment_IDs` but good for preliminary changes